src/ui/src/app/Service/employee-service.service.ts

```diff
@@ -0,0 +1,26 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable, inject } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EmployeeServiceService {
+
+  constructor() { }
+  private http = inject(HttpClient);
+  employeeId = 1;
```

Comment:
hard coded employeeId, please get it from the storage, private httpClient: HttpClient,